### PR TITLE
Update gitbook documentation for gitbook

### PR DESCRIPTION
**Pull request by lynparbone**
Update the gitbook docs to include a short description of the docker image with Graphviz for CircleCI.

### CODE REVIEW

File: docs/contribute/Documentation-and-Webpage.md [hunk 1]
```diff
@@ -60,3 +60,9 @@ If you ever need to publish manually:
 * When done: 
     * make sure that your changes to scenarioo/scenarioo are merged to develop 
     * on develop verify in `book.json` that the "develop"-version is configured as selected again!
+
+### Additional Requirements for PlantUML
+[PlantUML](https://plantuml.com/) is used to create and display UML diagrams in the documentation. To render the PlantUML diagrams, [Graphviz](https://graphviz.gitlab.io/about/) and Java 8 are required. 
+For these reasons, we've setup a special Docker image with the necessary tools installed and use it in the CircleCI job for automatically publishing the documentation.
```
Comment (bruderol) [inline docs/contribute/Documentation-and-Webpage.md]: perfect! Please add a note where the dockerfile is stored.

**Review by bruderol — CHANGES_REQUESTED** [review]
@lynparbone thank you very much! Perfect! Just one small note for improvement. Then you can merge it.

File: docs/contribute/Documentation-and-Webpage.md (hunk 1 above)
Comment (lynparbone) [reply docs/contribute/Documentation-and-Webpage.md]: Added the location, thanks for pointing that out!

**Review by bruderol — APPROVED** [review]
perfect!

**Pull request merged**